environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -69,6 +69,11 @@ public ValidationResult validate(EnvironmentValidationDto environmentValidationD
             return validationResult;
```

Comment:
This comment is for lines 61-64:
```
        AzureResourceGroupDto azureResourceGroupDto = azureParametersDto.getAzureResourceGroupDto();
        if (Objects.isNull(azureResourceGroupDto)) {
            return validationResultBuilder.build();
        }
```
So no further validations at all if no RG is given? This will make any DES checks below ineffective when using Multi-RG!

Reply:
I see, the validations in `AzureParameterValidator.java`  are for RG(until the encryption parameters were introduced). Updated the code, now first the encryption parameters will be validated and then RG validation.